File: Echo Client b0.0.6 SRC/src/main/java/hack/echo/client/features/impl/movement/NoJumpDelay.java
package hack.echo.client.features.impl.movement;

import hack.echo.client.event.EventSubscribe;
import hack.echo.client.event.impl.EventHandleInput;
import hack.echo.client.features.Feature;
import hack.echo.client.mixin.accessors.LivingEntityAccessor;
import hack.echo.client.utils.strings.Concat;
import hack.echo.client.features.Category;
import hack.echo.client.features.FeatureInfo;
import hack.echo.client.utils.strings.Concat;

public class NoJumpDelay extends Feature {

	public NoJumpDelay() {
		super(new FeatureInfo(
			Concat.of("No Jump Delay"),
			Concat.of("Removes jump cooldown"),
			Category.MOVEMENT
		));
	}

    @EventSubscribe
    public void onTick(EventHandleInput.Early event) {
        if (mc.player == null || mc.level == null) return;

        ((LivingEntityAccessor) mc.player).setNoJumpDelay(0);
    }
}

File: Grizzly b3 SRC/extracted/grizzlyclient/src/client/kotlin/onl/luka/grizzly/module/modules/movement/NoJumpDelay.kt
package onl.luka.grizzly.module.modules.movement

import onl.luka.grizzly.module.Module
import onl.luka.grizzly.mixin.client.LivingEntityAccessor
import net.minecraft.client.Minecraft

object NoJumpDelay : Module(
    name = "No Jump Delay",
    description = "Removes the jump cooldown. Works mostly for head hitters if you hold space and run down a 1x2 corridor",
    category = Category.MOVEMENT,
) {
    override fun onTick(client: Minecraft) {
        if (client.player == null) return
        (client.player as LivingEntityAccessor).setNoJumpDelay(0)
    }
}
